Source/WebCore/platform/graphics/SourceImage.cpp

```diff
@@ -37,6 +37,10 @@ SourceImage::SourceImage(ImageVariant&& imageVariant)
 {
 }
 
+SourceImage::SourceImage(const SourceImage&) = default;
+SourceImage::SourceImage(SourceImage&&) = default;
+SourceImage& SourceImage::operator=(const SourceImage&) = default;
+SourceImage& SourceImage::operator=(SourceImage&&) = default;
```

Comment:
Seems like these could just be kept in the header, but I guess the destructor's already here.

Reply:
I don't know any WebKit documents. But, this is a common C++ practice which is explained in C++ books.

Reply:
I know setting `default` for constructors and copy constructors is a thing, but I didn't expect it to be a solution for the problem of `SourceImage` having a `std::variant<..., Ref<ImageBuffer>, ...>` member variable with an `ImageBuffer.h` include. What's the connection?

Reply:
generated SourceImage's member functios calls ImageBuffer's member functions.
* ImageBuffer::ImageBuffer
* ImageBuffer::~ImageBuffer
* ImageBuffer::operator=
That's the reason ImageBuffer.h has to be incluced.